### PR TITLE
Update beaver to v0.2.2 and test using latest compilers

### PR DESCRIPTION
**Pull request by mathias-lang-sociomantic**
```
* submodules/beaver v0.1.2(f60fd6c)...v0.2.2(480e6f5) (18 commits)
  > gen-dockerfile: Fix docker repository and tag parsing
  > editorconfig: Fix indent for Makefiles
  > Mention wildcard support in README
  > bintray: Fix upload command-line arguments
  > Add a simple Makefile for easy local testing
  (...)
```

### CODE REVIEW

**Comment by mathias-lang-sociomantic** [comment]
Considering that all *proto won't be able to test with v2.071.x before they update to a branch including this P.R. should we aim for a patch release ?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+build/
```
Comment (nemanja-boric-sociomantic) [inline .gitignore]: Missing newline :-)

Comment (mathias-lang-sociomantic) [reply .gitignore]: Fixed ;)

**Review by mihails-strasuns-sociomantic — APPROVED** [review]

**Comment by mathias-lang-sociomantic** [comment]
So patch release or not ?

**Comment by mihails-strasuns-sociomantic** [comment]
Thanks!

**Comment by mihails-strasuns-sociomantic** [comment]
> So patch release or not ?

I would put it in minor by default unless this is needed for some project right now. Changes are quite harmless but numerous.

**Pull request merged**